packages/flutter/lib/src/semantics/semantics.dart

```diff
@@ -1774,6 +1775,15 @@ class SemanticsConfiguration {
     _setFlag(SemanticsFlag.isChecked, value);
   }
 
+  /// Whether the owning [RenderObject] is in a mutally exclusive group.
```

Comment:
nit: typo "mutally". Also this sentence just repeats the name of the field. Could you expand the description? For example, you could say that the owning RenderObject corresponds to UI that allows the user to pick one of several mutually exclusive options. The example below would back it up perfectly.

Reply:
Good idea. Done.